pkg/tstune/tuner.go

```diff
@@ -49,7 +49,7 @@ const (
 	fmtTunableParam = "%s = %s%s\n"
 	fmtLastTuned    = "timescaledb.last_tuned = '%s'"
 
-	lastTunedDateFmt = "2006-01-02 15:04"
+	lastTunedDateFmt = "2006-01-02 15:04:05.999999 -0700"
```

Comment:
not familiar with the code...when running, will this be replaced with a dynamic value that is the last tuned time?

Reply:
@solugebefola yup, go uses a fixed timestamp as a template to generate the actual output

Reply:
or does this describe the format?

Reply:
Yeah, as @JLockerman said, Go uses a fixed string template to describe how a particular time should be printed. So this is `YYYY-MM-DD hh:mm:ss.uuuuuu timezone`, or ISO8601

Reply:
Not sure it matters but I think postgres does not put a space between the nanos and the `-` for the timezone. Should we change just to be consistent?